test/language/block-scope/syntax/leave/verify-context-in-finally-block-1.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) Copyright 2011 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 13.1
+description: >
+    verify context in finally block 1
+---*/
+function f() {}
+
+try {
+  (function(x) {
+    try {
+      let x = 'inner';
+      throw 0;
+    } finally {
+      f();
+      assert.sameValue(x, 'outer');
+    }
+  })('outer');
+} catch (e) {}
```

Comment:
I think you need to re-throw here--otherwise, assertion errors will not be reported.